users/drashna/drashna.c

```diff
@@ -20,6 +20,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "rgb_stuff.h"
 
 userspace_config_t userspace_config;
+#if (defined(UNICODE_ENABLE) || defined(UNICODEMAP_ENABLE) || defined(UCIS_ENABLE))
+  #define DRASHNA_UNICODE_MODE UC_WIN
+#else
+  // set to 2 for UC_WIN, set to 4 for UC_WINC
+  #define DRASHNA_UNICODE_MODE 2
```

Comment:
`.gitlab-ci.yml` heck yes 👌

But this magic number here is asking for trouble. Are you doing it because the `process_unicode_common.h` never gets included if none of the Unicode methods are enabled? In that case, why do you even need a Unicode mode. I think this is a bad way of going about it.

Reply:
It's there because when unicode isn't set, I still want to be able to set the unicode method when resetting the EEPROM.  Check line 435. 

And yeah, the gitlab CI stuff is awesome, and you can configure a custom path for the yaml file, so I can add this and not affect anyone else's repo.

Reply:
Alright, you shouldn't need this once `UNICODE_SELECTED_MODES` is merged (as that will initialize it to your preferred input mode, when defined and as needed). For now though, this LGTM.